setup.py

```diff
@@ -0,0 +1,31 @@
+"""
+    dcm2niix-gear
+"""
+import setuptools
+
+VERSION = "1.2.2_1.0.20201102"
+
+with open("README.md", "r") as f:
+    LONG_DESCRIPTION = f.read()
+
+setuptools.setup(
+    name="dcm2niix-gear",
```

Comment:
Would rename as "fw-gear-dcm2niix". In gitlab that's how we have started enforcing this. The idea being to group all gear related packages under a common prefix.